src/natcap/invest/validation.py

```diff
@@ -165,9 +165,6 @@ def load_fields_from_vector(filepath, layer_id=0):
         A list of string fieldnames within the target layer.
 
     """
-    if not os.path.exists(filepath):
```

Comment:
Removing this so we don't have to handle remote vs. local paths. `gdal.OpenEx` below will raise an error if the file isn't found.